dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -239,12 +230,23 @@ USER root
 # We don't use npx playwright install-deps because that could pull a newer version
 # Install additional fonts as well
 RUN for iter in {1..10}; do \
-    apt-get update -y && \
-    $NODE_PATH/node/lib/node_modules/@elastic/synthetics/node_modules/.bin/playwright install-deps chromium && \
-    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends --yes \
-    fonts-noto \
-    fonts-noto-cjk && \
-    exit_code=0 && break || exit_code=$? && echo "apt-get error: retry $iter in 10s" && sleep 10; \
+        microdnf -y update && \
+        microdnf -y install fontconfig freetype cairo glib2 gtk3 pango xorg-x11-fonts-misc xorg-x11-fonts-Type1 \
+        at-spi2-atk atk at-spi2-core alsa-lib cups-libs dbus-libs libdrm mesa-libEGL mesa-libgbm nspr nss libX11 \
+        libX11-xcb libxcb libXcomposite libXdamage libXext libXfixes libXrandr libxkbcommon libxshmfence glib2 \
+        dbus-glib libicu mesa-libGL unzip iptables systemd && \
+        mkdir -p /usr/share/fonts/google-noto && \
+        curl -LO https://noto-website-2.storage.googleapis.com/pkgs/NotoSansCJKjp-hinted.zip && \
+        unzip NotoSansCJKjp-hinted.zip -d /usr/share/fonts/google-noto && \
+        rm -f NotoSansCJKjp-hinted.zip && \
+        microdnf -y remove unzip && \
+        curl -LO https://github.com/googlefonts/noto-fonts/raw/main/hinted/ttf/NotoSans/NotoSans-Regular.ttf && \
+        mv NotoSans-Regular.ttf /usr/share/fonts/google-noto && \
+        curl -LO https://github.com/googlefonts/noto-emoji/raw/main/fonts/NotoColorEmoji.ttf && \
+        mv NotoColorEmoji.ttf /usr/share/fonts/google-noto && \
+        fc-cache -fv && \
+        microdnf clean all && \
+        exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; \
```

Comment:
Was this tested using synthetics/monitors from the `complete` image?
If not, you can follow the steps from the description of this PR https://github.com/elastic/elastic-agent/pull/5475 (I did it for Wolfi-based complete image).

Reply:
Thanks for the proposal @rdner that was quite informative and re-assuring that synthetics work here are my proof of testing 🙂 

<img width="2933" alt="Screenshot 2025-01-07 at 7 33 30 PM" src="https://github.com/user-attachments/assets/2d6dcc4c-6596-46d3-8e68-9705f666fd1b" />
<img width="2920" alt="Screenshot 2025-01-07 at 7 33 12 PM" src="https://github.com/user-attachments/assets/d1672d5e-a7f2-46bc-b159-062963d6029a" />

Reply:
thanks for the proposal @rdner this is quite helpful and actually re-assuring that this PR won't cause any issues with synthetics. Here is my proof of testing exactly what you tested in linked PR above 🙂 

<img width="2926" alt="Screenshot 2025-01-07 at 9 30 25 PM" src="https://github.com/user-attachments/assets/4124d2bc-c308-48ec-868c-3fc2156f27a7" />
<img width="2931" alt="Screenshot 2025-01-07 at 9 32 00 PM" src="https://github.com/user-attachments/assets/c73f00f4-db5b-4077-92b1-500a7054d018" />
<img width="2927" alt="Screenshot 2025-01-07 at 9 32 10 PM" src="https://github.com/user-attachments/assets/f3fce32b-2546-4ae5-b1fd-93db15a0f19a" />